### DISCUSSION TITLE
examples dict for query parameter showing warning: Could not render n, see the console

### DISCUSSION THREAD

**Discussion by bluebrown** [fastapi/fastapi#4266] [Questions] [answered]
Hi, I am trying to provide [multiple examples][1] for a query parameter. 

```python
def handler(
    status: str = Query(
        None,
        title="Status",
        description=f"Comma separated list. Possible values are: {', '.join(status_list)}.",
        examples={
            "single": choice(status_list),
            "list": ",".join(sample(status_list, 3)),
        },
    ),
):
...
```

But it's not showing properly in the docs page. There is a message saying:

> 😱 Could not render n, see the console.

When I check the console, I see the below stacktrace.

```
root-injects.jsx:94 
TypeError: e.get is not a function
at examples-select.jsx:130
at immutable.js:3016
at immutable.js:2699
at St.__iterate (immutable.js:2206)
at zt.__iterate (immutable.js:2698)
at r.Xt.r.__iterateUncached (immutable.js:3015)
at pe (immutable.js:604)
at r.J.__iterate (immutable.js:274)
at r.forEach (immutable.js:4381)
at immutable.js:2651
```

When I use the 2 examples individually as a single example with the keyword argument `example`, I don't get an error.

  [1]: https://fastapi.tiangolo.com/tutorial/schema-extra-example/

**Comment by simondale00** [comment, marked as answer]
Hey @bluebrown. You'll want to change your `examples` kwarg as follows:

```
examples={
    "single": {
        "value": choice(status_list),
    },
    "list": {
        "value": ",".join(sample(status_list, 3)),
    }
}
```

`examples` expects a nested dictionary, where at the root your keys are names for each example, like "single" or "list". But the value of that key must be another dictionary containing specific keywords. [This section](https://fastapi.tiangolo.com/tutorial/schema-extra-example/#body-with-multiple-examples) of the documentation discusses the expected format for `examples`.
